src/main/java/Exercise1.java

```diff
@@ -0,0 +1,22 @@
+public class Exercise1 {
+
+    public int ucln(int a, int b) {
+        if (a == 0 || b == 0)
+            return (a + b);
+        else {
+            while (a != b) {
+                if (a > b)
+                    a -= b;
+                else
+                    b -= a;
+            }
+        }
+        return a;
+    }
+
+    public int bcnn(int a, int b) {
+        int x;
```

Comment:
Khi khai báo biến, nên hạn chế việc khai báo một nơi, sử dụng một nẻo. Như trong tình huống này, hoàn toàn có thể vừa khai báo vừa khởi tạo giá trị phù hợp

Reply:
Done!